### PR TITLE
fix panic condition in parseCompositeID

### PR DESCRIPTION
**Pull request by talbright**
Addresses #146

### CODE REVIEW

File: heroku/provider.go [hunk 1]
```diff
@@ -113,9 +114,13 @@ func buildCompositeID(a, b string) string {
 	return fmt.Sprintf("%s:%s", a, b)
 }
 
-func parseCompositeID(id string) (string, string) {
+func parseCompositeID(id string) (p1 string, p2 string) {
 	parts := strings.SplitN(id, ":", 2)
-	return parts[0], parts[1]
+	p1 = parts[0]
```
[the remark this answers is not in the document]
Comment (mars) [reply heroku/provider.go]: I agree that an informative error would be ideal. As clear as possible, like:

“Import ID requires two parts, a composite ID separated by colon, like `id1:id2`.”

[the remark this answers is not in the document]
Comment (mars) [reply heroku/provider.go]: I agree that an informative error would be ideal. As clear as possible, like:

“Import ID requires two parts, a composite ID separated by colon, like `id1:id2`.”

[the remark this answers is not in the document]
Comment (mars) [reply heroku/provider.go]: I agree that an informative error would be ideal. As clear as possible, like:

“Import ID requires two parts, a composite ID separated by colon, like `id1:id2`.”

[the remark this answers is not in the document]
Comment (talbright) [reply heroku/provider.go]: Laziness is a virtue ya know Zzzzzzzzzzzzzz

**Comment by talbright** [comment]
@davidji99 used err message per your suggestion, can you take another look?

**Pull request merged**